src/components/FavButton/FavButton.js

```diff
@@ -1,14 +1,12 @@
 import React from 'react';
-import './FavButton.css';
+import './FavButton.css'
 
-const FavButton = ({ favorites }) => {
-  return (
+const FavButton = ({favorites})=>{
+  return(
     <div>
-      <button className='btn-favorite'>
-        Favorites {favorites.length}
-      </button>
+      <button> Favorites {favorites.length} </button>
```

Comment:
This change breaks tests because the className is required for the tests. Also, the removal of the semi-colons kicks back ESLint errors.